src/applications/personalization/profile-2/tests/components/ProfileHeader.unit.spec.jsx

```diff
@@ -6,14 +6,25 @@ import ProfileHeader from '../../components/ProfileHeader';
 
 const fakeStore = {
   getState: () => ({
-    user: {
-      profile: {
+    vaProfile: {
+      hero: {
         userFullName: {
           first: 'Johnnie',
           middle: 'Leonard',
           last: 'Weaver',
         },
       },
+      militaryInformation: {
+        serviceHistory: {
+          serviceHistory: [
+            {
+              branchOfService: 'Army',
+              beginDate: '1/2/2004',
```

Comment:
might not matter, but to better reflect reality these dates should be like: 2004-2-1